hacks/roadcraft-nvidia-570-stable/pyroveil.json

```diff
@@ -2,7 +2,151 @@
 	"version": 2,
 	"type": "pyroveil",
 	"matches": [
-		{ "opStringSearch": "4935cbe43b8f9458", "action": { "glsl-roundtrip": true } }
+		{ "opStringSearch": "4935cbe43b8f9458", "action": { "glsl-roundtrip": true } },
```

Comment:
... what?

Reply:
It's the added comma, I can rearrange the lines if you want?

Reply:
It's quite probably including shaders that don't need to be, though I'm not sufficiently knowledgeable about the process for pinning down the problematic shader. If you could provide instructions for identifying exactly which shaders need to be included, I (and many others) would very much appreciate it.

I generated this file by running with `VKD3D_SHADER_DUMP_PATH` on both my main machine with Nvidia GPU and on my other machine with an AMD GPU. Then diffed the two lists of files and took the hashes which were missing from the Nvidia dump. I'm sure there's a better less clumsy way than this but I don't know what it is.